README.md

```diff
@@ -113,7 +120,7 @@ If the balance is less than the `threshold`, you want to set a `openForWithdraw`
 
 ⏳ _"Time Left"_ will only update if a transaction occurs. You can see the time update by getting funds from the faucet button in navbar just to trigger a new block.
 
-If the `Staker UI` tab _"Time Left"_ does not update correctly when a transaction occurs, try switching to other tab and come back to it.
+If the `Staker UI` tab _"Time Left"_ does not update correctly when a transaction occurs, try switching to another tab and come back.
```

Comment:
Maybe we can get rid of this sentence, I think is working fine after PR #25

Reply:
Agreed.  It has been removed.